sdk/spring/scripts/compatibility_insert_dependencymanagement.py

```diff
@@ -45,6 +47,18 @@ def add_dependency_management_for_all_poms_files_in_directory(directory, spring_
                 add_dependency_management_for_file(file_path, spring_boot_dependencies_version, spring_cloud_dependencies_version)
 
 
+# Downgrade mysql connector when Spring Boot version is 2.5.14
```

Comment:
My thought is when we are using 2.5.14, we can skip the process of `removing/deleting version` from this artifact, so we still use the current version, instead of the version in 2.5.14 for MySQL.

Reply:
We can ignore it because the new artifact id does not exist in the spring-boot-dependencies 2.5.14, so its version can be removed.